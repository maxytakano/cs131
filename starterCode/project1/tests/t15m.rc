function : void main()
{
  int****** ptr0 = nullptr;
  int* ptr1 = nullptr;
  int* ptr2 = ptr1;
  float* ptr3 = ptr0;
  int* ptr4 = ptr0;
}
