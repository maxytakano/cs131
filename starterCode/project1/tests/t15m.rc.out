Error, "/tmp/cs131_testrunner_ZHTaux/file.rc", line 6: 
  Initialization value of type int****** not assignable to constant/variable of type float*.
Error, "/tmp/cs131_testrunner_ZHTaux/file.rc", line 7: 
  Initialization value of type int****** not assignable to constant/variable of type int*.
Compile: failure.
